tests/simple/simple.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
I think the line style comments are more common, but this is a super nit.

Reply:
When the world starts running short of slash characters, you'll regret having squandered yours on gaudy comment banners. :-)  Leaving as-is, since I've seen standard lib go files with this style for long package comments.  I should also write a README.md file.